documentation/experimental_features.md

```diff
@@ -8,6 +8,30 @@ API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
 around breaking behavior where possible. 
 
+## Configuration policies and `bolt policy` subcommand
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+Configuration policies are public classes that can be assigned directly to one
+or more nodes via the Bolt command line. Policies are Puppet code stored in .pp
+files.
+
+Policies can be accessed through the `policy` subcommand, with `apply`, `new`
+and `show` options. The default location for a policy created through Bolt is
+`<Bolt project directory>/manifests/<policyname>.pp`. Bolt creates an empty
+class to fill in with the desired configuration.
+
+Once a policy is created it can be applied to target nodes via `bolt policy
+apply` and available policies can be listed via `bolt policy show`.`bolt policy
+apply` accepts a comma-separated list of policy names to apply, or a single
+policy name to apply to a list of one or more targets. Use '-h' with any of the
+policy subcommand options for more detail.
```

Comment:
"Bolt will create a single line of Puppet code `include #{policies.join(,)}` that will be compiled and applied to the provided targets, and will log that code at the debug level."
I think this will clarify some questions that people who are more experienced may have around how classes are applied, are they applied in order, etc.